src/systemd/sd-id128.h

```diff
@@ -34,6 +34,7 @@ union sd_id128 {
 };
 
 #define SD_ID128_STRING_MAX 33
+#define SD_ID128_UUID_STRING_MAX 37
```

Comment:
so far we kept id128_to_uuid_string() out of the public headers, hence it would be weird to place SD_ID128_UUID_STRING_MAX there, which is only used for it. Maybe define that in id128-util.h too for now (and thus drop the SD_ prefix which indicates exported APIs to us)